src/app/components/ScheduleSummary/ScheduleSummary.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import './ScheduleSummary.css';
+import { Glyphicon } from 'react-bootstrap';
+
+export enum StatusEnum {
+    GOOD,
+    BAD,
```

Comment:
Did we want our convention to be All caps for Enums?  Is that what we did with weekdays?

Reply:
For weekdays we used 'Mon', 'Tues', ...

Reply:
Ok, I leave it to you. If you like all caps, then roll with it 👍